documentation.go

```diff
@@ -201,28 +201,32 @@ func (c *documentationCommand) writeDocs(folder string, superCommands []string,
 		if !printDefaultCommands && isDefaultCommand(name) {
 			continue
 		}
+
 		commandSeq := append(superCommands, name)
-		target := fmt.Sprintf("%s.md", strings.Join(commandSeq[1:], "_"))
-		target = strings.ReplaceAll(target, " ", "_")
-		target = filepath.Join(folder, target)
 
-		f, err := os.Create(target)
-		if err != nil {
-			return err
-		}
-		writer := bufio.NewWriter(f)
-		formatted := c.formatCommand(ref, false, commandSeq)
-		_, err = writer.WriteString(formatted)
-		if err != nil {
-			return err
+		sc, isSuperCommand := ref.command.(*SuperCommand)
+		if !isSuperCommand || (isSuperCommand && !sc.FlagDocumentationHidden) {
```

Comment:
Use `continue` to avoid indentation
```go
if X {
  Y...
}
```
===>
```go
if !X {
  continue
}
Y...
```

Reply:
This should also reduce the diff

Reply:
That's not how this code works. Also, we're attempting to keep the order of the markdown. Changing the markdown order is a breaking change, so I don't want to do that.